DataFormats/ForwardDetId/test/testHGCDetId.cc

```diff
@@ -0,0 +1,129 @@
+#include "DataFormats/ForwardDetId/interface/HGCScintillatorDetId.h"
+#include "DataFormats/ForwardDetId/interface/HGCSiliconDetId.h"
+#include "DataFormats/DetId/interface/DetId.h"
+
+#include <cmath>
+#include <iomanip>
+#include <iostream>
+#include <map>
+#include <string>
+
+void testCell(int type) {
+
+  int    N  = (type == 0) ? 12 : 8;
```

Comment:
Use defined constants `HGCalFineN` and `HGCalCoarseN` here

Reply:
This I shall do